### PR TITLE
More extensive travis testing

### LINKED ISSUE #7: Smoke Tests for CLI
Introduce smoke tests for new releases to check that it is even possible to initialize and install the new release.

### PR DESCRIPTION
**Pull request by hfjn**
Took me some attempts. Also closes #7.

### CODE REVIEW

File: esque/cli/commands.py [hunk 1]
```diff
@@ -14,16 +15,22 @@
...
-from esque.config import PING_TOPIC, Config, PING_GROUP_ID
+from esque.config import PING_TOPIC, Config, PING_GROUP_ID, config_dir, sample_config_path, config_path
 from esque.consumergroup import ConsumerGroupController
 from esque.errors import ConsumerGroupDoesNotExistException, ContextNotDefinedException, TopicAlreadyExistsException
 from esque.topic import TopicController
 
 
-@click.group(help="(Kafka-)esque.")
+@click.group(help="esque - an operational kafka tool.", invoke_without_command=True)
+@click.option("--recreate-config", is_flag=True, default=False, help="Overwrites the config with the sample config.")
```
Comment (MrTrustworthy) [inline esque/cli/commands.py:25]: i'd personally prefer to have this as command, not as an option for the base `esque` command

File: .travis.yml [hunk 2]
```diff
@@ -45,12 +45,50 @@ matrix:
...
-    tags: true
-    condition: $TRAVIS_PYTHON_VERSION == '3.6'
-    repo: real-digital/esque
-  password:
-    secure: L7lAviuPS2FijT/t01t0SVHuxfl3/6jcSU3anWkcJdWsa9ktujvdJ6QoX0VE+niUyAwnJ6HJUNWloG1EuuRPiaVO+vIU0t39oKtP/3KKGz6zQPme4p0tpgxvPCjxzBxlaj3zs+3KlfqT7ig5EllgLcAkyCmMpvYgpPY8pCsIY/ipZ+zzQvrA9glCQex9LPhdWH7yaHgx9eGM/dQumn9J/+/CtEkbxLpmw/WHRahCgoODFBMQjTQsVleiHZqedx7gQsSD7lqbdsUkBujawdYIrGnBVIjMdqXT7nQPbKmntKf+5GMcPdFaOqgciQYYgIRLixP1JWPoHzTvsmp0Jfmd8/4u4Eh/zLOKjdsdlNk/X2hNYae+YvKXMSwYqY33cDezsIpkvbz+Vfu3Ch97c2pmp3pSTTkcxKMqnoQJ3aqZENveCPOgfb3PSrzwYBKENaFLZ45l9eU1CPgCNNa7oX/CA1+kb1wY6AMCLW4ZxOe4I3EoUpNyn4lOIEkeOsHIB5d9DJvphd5UrFgUdnADzsnAsr38Gq+Haz9YmaXscUIRvQR27HDqCwfsSn9XPkdwImIjzhPrgM5kGXm0UPsf7nGZiYwEuB2fDTji4YneDhCF4aCKzr7Txlo1BtWp5ue6kbjCARBkwgCO/uHkl8AYos/tUmZe6vl6p3kwYf8sl3i9bOs=
+  - provider: pypi
+    user: real.digital
+    on:
+      tags: true
+      condition: $TRAVIS_PYTHON_VERSION == '3.6'
```
Comment (MrTrustworthy) [inline .travis.yml:91]: So we test the upload and everything with PyPi on Python3.7, but then we upload on the 3.6 pipeline? Is there a specific reason for that?

**Review by MrTrustworthy — APPROVED** [review]
Looks like developing & debugging this was pretty annoying :D

File: .travis.yml (hunk 2 above)
Comment (hfjn) [reply .travis.yml:91]: let’s call it an accident :D

File: esque/cli/commands.py (hunk 1 above)
Comment (hfjn) [reply esque/cli/commands.py:25]: I don’t. But that’s up for discussion

**Pull request merged**